lib/trace-writer.js

```diff
@@ -120,14 +120,10 @@ TraceWriter.prototype.flushBuffer_ = function(projectId) {
   // Privatize and clear the buffer.
   var buffer = this.buffer_;
   this.buffer_ = [];
+  var joined = buffer.join();
+  buffer = '[' + joined.slice(0, joined.length -1) + ']';
```

Comment:
I am not sure if I understand why the slice is necessary. Doesn't it remove the last `}`? Also, need space before `1`.

For example:
```
var a = {foo: 'bar'};
[a,a,a,a].map(JSON.stringify).join()
→ "{"foo":"bar"},{"foo":"bar"},{"foo":"bar"},{"foo":"bar"}"
```
`slice` as used above will remove the last `}`, no?